### ISSUE TITLE
Styling for App

### ISSUE THREAD

**Issue by Ipaye** [lauragift21/MovieSearchApp#3]
It seems like there isnt any styling for this node app

**Comment by lauragift21** [comment]
Yes it's just Basic I will start working on the styling or if you don't mind you can help with that.

**Comment by Ipaye** [comment]
Oh, okay. i can try to do that
